Dockerfile

```diff
@@ -47,12 +47,14 @@ VOLUME [ "/opt/system/tmp/cache/", \
          "/opt/system/node_modules", \
          "/opt/system/assets/jspm_packages", \
          "/opt/system/public/assets", \
+         "/opt/system/public/packs-test", \
          "/root/.jspm", "/home/ruby/.luarocks" ]
 
 ADD . ./
 ADD config/examples/*.yml config/
 # Needed for Sphinx ODBC
 ADD config/oracle/odbc*.ini /etc/
+COPY config/webpacker.ci.yml config/webpacker.yml
```

Comment:
Can we have a directory `config/ci` instead?

Reply:
Maybe in the Dockerfile we can do
```Dockerfile
ADD config/examples/**/*.yml config/
```

but in local keep doing

```shell
cp config/examples/*.yml config/
```

Reply:
We could but I do not know what to put there?

Reply:
`condif/examples/ci/webpacker.yml`

Reply:
Just this file? I do not see the point here

Reply:
I see now. `webpacker.ci.yml` is not in `config/examples`.